teacher.rb

```diff
@@ -0,0 +1,11 @@
+# Class Teacher file.
+class Teacher < Person
+  def constructor(specialization)
```

Comment:
- [ ] Kindly import your **`person`** file here so that the Teacher class can inherit from the Person class